conans/paths/package_layouts/package_cache_layout.py

```diff
@@ -117,6 +124,24 @@ def package_revision(self, pref):
         tm = metadata.packages[pref.id].time if metadata.packages[pref.id].time else None
         return metadata.packages[pref.id].revision, tm
 
+    def conan_builds(self):
+        builds_dir = self.builds()
```

Comment:
below two functions are identical except the first line (builds vs packages)

Reply:
👍 , but this simplification will go in a different PR, we have to review all the code in `SimplePaths`, `ClientCache` and `layouts`, there are duplicates, code that is no longer needed,...